main.go

```diff
@@ -29,8 +29,7 @@ func main() {
 
 // Run is a function so that TestMain can execute it
 func Run() {
-	logging.Get().Infof("Skuid CLI Version %v", VERSION_NAME)
```

Comment:
@gzapatas Sorry to take back my approval; everything works as expected but I did wonder if it was intentional that you took this out? Personally I think it's a good idea to show the user what version they're on whenever they do a retrieve / deploy / whatever

Reply:
@adewaal I removed it because "skuid-cli --version" is showing the version twice.
![image](https://user-images.githubusercontent.com/30167939/217597677-4464fd1c-8dc7-4895-8833-80c10f390668.png)

Reply:
Okay, that makes sense. Is there a way to log it just when retrieving / deploying then?

Reply:
I can add that line in the RunE handler of each command to avoid showing the version log and the version sent by the --version flag.